solutions/tests/test_length_of_longest_substring.py

```diff
@@ -0,0 +1,43 @@
+import unittest
+from solutions.length_of_longest_substring import Solution
+
+
+class TestLengthOfLongestSubstring(unittest.TestCase):
+    """Unit tests for lengthOfLongestSubstring function."""
```

Comment:
Should match the name of the function
```suggestion
    """Unit tests for length_of_longest_substring function."""
```

Reply:
noted!